src/settings-ui/Settings.UI/SettingsXAML/Flyout/LaunchPage.xaml.cs

```diff
@@ -135,6 +136,11 @@ private void ModuleButton_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private void UpdateDetailsBtn_Click(object sender, RoutedEventArgs e)
+        {
+            App.OpenSettingsWindow(typeof(GeneralPage), true);
```

Comment:
So, this is navigating, not initiating an actual action.
https://learn.microsoft.com/windows/apps/design/controls/infobar#add-an-action-button

Reply:
Yes it is a navigation button use case. What is your request here. If you like an other text please suggest one.

Reply:
Is this related to the label in the button "Install"?

Currently it's "Learn more" which I think is vague but covers what it does. I think "Install" is OK as well, although it requires yet another click.

"Get it" could be an alternative, see W11 update dialog:

![image](https://github.com/microsoft/PowerToys/assets/9866362/b22e698e-c3cb-4e9e-8e0b-fe1ba460bc1c)

Reply:
@niels9001 
Yes. I have no better text idea. And I see no difference between "Get it" and "Install". I think the best is to have only the text "Update available" as button. But this needs a bigger xaml rework as a infobar without title or text looks misaligned.

Reply:
Short version: Buttons are for immediate action, Hyperlinks for navigation. Right?

Reply:
I will try to ideas tomorrow and let you know about the results.

Reply:
@jay-o-way, @niels9001 
Are you both okay with this?
![image](https://github.com/microsoft/PowerToys/assets/61519853/e588af0c-ff02-43c8-9a28-0c2863de75db)
![image](https://github.com/microsoft/PowerToys/assets/61519853/d0b31678-5a27-4194-898d-12ed58913707)

@niels9001 
I tried an infobar without title and message but then the icon is misaligned.
![image](https://github.com/microsoft/PowerToys/assets/61519853/162a0da1-94bd-4ee4-85e0-15bbc46d46db)